packages/fiber/src/core/hooks.tsx

```diff
@@ -51,6 +52,14 @@ export function useFrame(callback: RenderCallback, renderPriority: number = 0):
   return null
 }
 
+export function useUpdate(callback: UpdateCallback, stage: string = 'update') {
```

Comment:
stage can only be a specific string, this is where the enum idea would help more in TS codebases.

Reply:
Also be good to add JSDoc notes like we have on other hooks